signac/contrib/project.py

```diff
@@ -1493,8 +1505,10 @@ def export_to(self, target, path=None, copytree=None):
 
         See Also
         --------
-        :meth:`~signac.Project.import_from` : Previously exported or non-signac
-            data spaces can be imported with.
+        :meth:`~signac.Project.import_from` :
+            Previously exported or non-signac data spaces can be imported.
+
```

Comment:
I think we need another line here to keep it out of the `See Also` box

Reply:
fixed!